test/webhook-proxy.test.ts

```diff
@@ -1,13 +1,19 @@
-const express = require('express')
+import express from 'express'
+import { Server } from 'http'
+import nock from 'nock'
+import { logger } from '../src/logger'
+import { createWebhookProxy } from '../src/webhook-proxy'
+
+// tslint:disable-next-line:no-var-requires
 const sse = require('connect-sse')()
```

Comment:
For the sake of simplicity (or maybe laziness lol), I didn't convert this require statement to a TypeScript import, since this module doesn't have types. Happy to go back and convert this properly - what's your preference on supplying proper types for a third-party, test-only dependency?

Reply:
As you say, it's a third-party, test-only dependency, so I'm fine with this as is!